metadata/src/lib.rs

```diff
@@ -395,6 +395,23 @@ pub enum StorageEntryType {
     },
 }
 
+impl StorageEntryType {
+    /// returns the type of the value.
```

Comment:
small nit: Could start with uppercase here to be consistent with docs / fn below